services/contentful.js

```diff
@@ -18,20 +18,25 @@ module.exports.initClients = (options) => {
   const applicationName = `the-example-app.nodejs/${version}`
 
   const config = options || {
-    spaceId: process.env.CF_SPACE_ID,
-    deliveryToken: process.env.CF_DELIVERY_TOKEN,
-    previewToken: process.env.CF_PREVIEW_TOKEN
+    spaceId: process.env.CONTENTFUL_SPACE_ID,
+    deliveryToken: process.env.CONTENTFUL_DELIVERY_TOKEN,
```

Comment:
you can make the name generic here `CONTENTFUL_TOKEN` since the token will change based on the `host`